botCommands/points.js

```diff
@@ -4,12 +4,29 @@ const { registerBotCommand } = require('../botEngine.js');
 
 axios.defaults.headers.post.Authorization = `Token ${config.pointsbot.token}`;
 
-function getUserIdsFromMessage(text, regex) {
+function getUserIdsFromMessage(client, author, guild, text, regex, authorMember, channel) {
   const matches = [];
   let match = regex.exec(text);
+
   while (match !== null) {
-    matches.push(match[1].replace('!', ''));
-    match = regex.exec(text);
+    if (match[2] === '?++') {
+      let isAdmin = false;
+      authorMember.roles.cache.forEach((value) => {
+        if (['core', 'maintainer', 'admin'].includes(value.name)) {
+          isAdmin = true;
+        }
+      });
+
+      if (isAdmin) {
+        matches.push([match[1].replace('!', ''), 2]);
+      } else {
+        channel.send('Only admin users can give double points!');
```

Comment:
Can we change this to "maintainers or core members"?

Reply:
Done